tasks/46_running-average/running-average.js

```diff
@@ -0,0 +1,8 @@
+const arr = [];
```

Comment:
What if I will use this function twice?

Reply:
i need to replace `arr` from global scope to  function scope to avoid adding new elements to old array